app/src/main/java/de/tum/in/tumcampusapp/api/TUMCabeClient.java

```diff
@@ -83,23 +83,6 @@ private TUMCabeClient(final Context c) {
         builder.client(Helper.getOkClient(c));
         service = builder.build()
                          .create(TUMCabeAPIService.class);
-
-        /*
-        TODO port the error handler to Retrofit 2
-        ErrorHandler errorHandler = new ErrorHandler() {
-            @Override
-            public Throwable handleError(RetrofitError cause) {
-                Throwable t = cause.getCause();
-                if (t instanceof SSLPeerUnverifiedException) {
-                    //TODO show a error message
-                    //Toast.makeText(context, t.toString(), Toast.LENGTH_LONG).show();
-                }
-
-                //Return the same cause, so it can be handled by other activities
-                return cause;
-            }
-        }; */
-
```

Comment:
Has the error handler been implemented?
If not, please leave the TODO

Reply:
As of retrofit 2, network errors throw a normal ioexception. There is no need to have a central error handler, as previously.

If we really do want to logg this somewhere central, you would need to add an interceptor to the OkHttpClient - we already have one there.